### DISCUSSION TITLE
Support for Apple Homekit Secure Video

### DISCUSSION THREAD

**Discussion by consolecwby** [blakeblackshear/frigate#11362] [Ideas]
So the functionality of streaming cameras to homekit works great, especially with other members of my house. However i can only stream id like to be able to also have a backup cloud recording that i am already paying for with the Apple HomeKit Secure video feature.  I have just been using Scrypted now to get HSKV to work but would rather use frigate and only as my local video storage and cloud storage integrator. 
Also know go2rtc is needed for this but  frigate has the motion detection capabilities needed for HKSV to work.

**Comment by nin9s** [comment]
ooc: currently using the scrypted rebroadcasted stream for frigate object detection but would like to get rid of scrypted as its only purpose in my case is hksv **recording**.
HomeKit viewing via frigate works perfect but afaik recording is impossible.

will this ever be considered to be worked on?

**Reply by consolecwby** [reply]
I finally switched to frigate NVR since I couldn't get the rebroadcast to work, couldn't figure out how to configure things correctly.

**Reply by nin9s** [reply]
but with frigate only you do not have hksv recordings do you?
